compiler/compile/OMRNonHelperSymbols.enum

```diff
@@ -399,7 +399,12 @@
     */
    J9JNIMethodIDvTableIndexFieldSymbol,
 
-   OMRlastPrintableCommonNonhelperSymbol = J9JNIMethodIDvTableIndexFieldSymbol,
+   /** \brief
+    * This symbol represents the defaultValue field in J9Class.flattenedClassCache
```

Comment:
Is there a possibility some downstream project other than OpenJ9 will want to use this symbol?  If so, I wonder whether the description could be made a little more generic - maybe "This symbol is used to access the default instance associated with a class, if any exists"

Reply:
Updated in 6c372db